### PR TITLE
Fix -v option and debug error

### PR DESCRIPTION
**Pull request by barakmich**

### CODE REVIEW

File: graph/proto/helpers.go [hunk 1]
```diff
@@ -10,10 +10,10 @@ import (
 // GetNativeValue returns the value stored in Node.
 func (m *NodeData) GetNativeValue() quad.Value {
 	if m == nil {
-		return nil
+		return quad.Raw("")
```
Comment (dennwc) [inline graph/proto/helpers.go]: Is this change necessary? Might break something.

Comment (barakmich) [inline graph/proto/helpers.go]: Fair enough. Fixed it the other way.

File: graph/bolt/iterator.go [hunk 2]
```diff
@@ -283,9 +283,18 @@ func (it *Iterator) Size() (int64, bool) {
 }
 
 func (it *Iterator) Describe() graph.Description {
+	name := ""
+	nameOf := it.qs.NameOf(&Token{
+		nodes:  true,
+		bucket: it.bucket,
+		key:    it.checkID,
+	})
+	if nameOf != nil {
```
Comment (dennwc) [inline graph/bolt/iterator.go]: `quad.StringOf` might help :)

File: graph/proto/helpers.go [hunk 3]
```diff
@@ -10,7 +10,7 @@ import (
 // GetNativeValue returns the value stored in Node.
 func (m *NodeData) GetNativeValue() quad.Value {
 	if m == nil {
-		return quad.Raw("")
+		return nil
 	} else if m.Value == nil {
 		if m.Name == "" {
 			return quad.Raw("")
```
Comment (dennwc) [inline graph/proto/helpers.go]: should also be reverted
